### DISCUSSION TITLE
How to use Proxy without CONNECT in HTTP

### DISCUSSION THREAD

**Discussion by voluntas** [ninenines/gun#316] [Q&A] [answered]
I am aware that Gun always sends a CONNECT method in Proxy behavior for both HTTP and HTTPS, is this correct?

Does Gun have a mechanism to send absolute paths without sending a CONNECT method when using HTTP?
I followed the source code all the way through and couldn't find it.

**Comment by voluntas** [comment]
It worked well by specifying Url in the part that specifies the Path. I will post a simple sample later.

**Reply by essen** [reply, marked as answer]
Right it won't check that a path is indeed a path so you can send a URI there if needed.
